web-apis/chrome/unified-plan/index.md

```diff
@@ -34,21 +34,28 @@ Unified Plan is currently developed, and the flag for experimentation has been a
 If you run Chrome with “--enable-blink-features=RTCUnifiedPlan”, you will have access to the “sdpSemantics” feature described above, and can start testing with Unified Plan.
 
 ### Phase 2: Make the API feature generally available
-Planned for M68 (beta June 2018).
+Released in M69 (beta August 2018, stable September 2018)
```

Comment:
The following comment belongs to the line above "The Phases" but github won't let me comment outside of the touched parts:

You can add that this can be controlled from chrome://flags as the feature "WebRTC: Use Unified Plan SDP Semantics by default" as of M71.

Reply:
Added!